fury/material.py

```diff
@@ -99,3 +113,130 @@ def _create_mesh_material(
         )
     else:
         raise ValueError(f"Unsupported material type: {material}")
+
+
+def _create_points_material(
+    *,
+    material="basic",
+    color=(1.0, 1.0, 1.0),
+    size=4,
+    map=None,
+    aa=True,
+    mode="vertex",
+    opacity=1.0,
+    enable_picking=True,
+):
+    """
+     Create a points material.
+
+     Parameters
+     ----------
+     material : str, optional
+         The type of material to create. Options are 'baisc' (default),
```

Comment:
Let's not create two different materials for the points and have one more if with marker and use the same method for both actors internally. Also the spelling for basic is incorrect in the doc.